raiden-ts/src/messages/utils.ts

```diff
@@ -26,13 +26,17 @@ const CMDIDs: { readonly [T in MessageType]: number } = {
   [MessageType.WITHDRAW_EXPIRED]: 17,
   [MessageType.PFS_CAPACITY_UPDATE]: -1,
   [MessageType.PFS_FEE_UPDATE]: -1,
+  [MessageType.MONITOR_REQUEST]: -1,
 };
 
 // raiden_contracts.constants.MessageTypeId
 export enum MessageTypeId {
   BALANCE_PROOF = 1,
+  BALANCE_PROOF_UPDATE = 2,
   WITHDRAW = 3,
+  COOP_SETTLE = 4,
```

Comment:
This was added for the sake of completion right? it is not used anywhere even in the python client.

Reply:
Yes, exactly, but we may want to add this feature at some point in the future, seems supported by the SC